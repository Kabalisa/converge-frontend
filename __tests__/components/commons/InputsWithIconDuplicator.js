import React from 'react';
import { shallow } from 'enzyme';
import InputsWithAddIcons from '../../../src/components/commons/InputAddIcon/index';

describe.only('Setup component', () => {
  const wrapper = shallow(<InputsWithAddIcons />);

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('Should add another input with plus icon on the page', () => {
    wrapper.instance().onAddInput();
    expect(wrapper.state('displayInputs')).toHaveLength(2);
  });
});
