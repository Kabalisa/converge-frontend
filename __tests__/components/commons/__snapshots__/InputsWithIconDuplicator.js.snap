// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input With Plus Icon component should render properly 1`] = `
<Fragment>
  <SingleInputWithAddIcon
    inputPlaceholder="Building"
    pressed={[Function]}
  />
</Fragment>
`;
